jadex.properties

```diff
@@ -1,4 +1,4 @@
-#Sun Oct 20 15:10:50 CEST 2019
```

Comment:
Este archivo que crea jadex mejor que no lo incluyáis en los commits, porque solo cambia la fecha y no tiene sentido